pycloudlib/ec2/cloud.py

```diff
@@ -161,7 +161,10 @@ def _get_name_for_image_type(self, release: str, image_type: ImageType, daily: b
             return f"ubuntu-pro-server/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
 
         if image_type == ImageType.PRO_FIPS:
-            return f"ubuntu-pro-fips*/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
+            return f"ubuntu-pro-fips-server*/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
+
+        if image_type == ImageType.PRO_FIPS_UPDATES:
+            return f"ubuntu-pro-fips-updates-server*/images/{disk_type}/ubuntu-{release}-{release_ver}-*"
```

Comment:
likewise, you can drop the `*` from `ubuntu-pro-fips-updates-server*`

Reply:
done